File: launchdarkly/php/src/GetBigSegmentStoreIntegrationExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("Authorization", "YOUR_API_KEY");

try {
    $response = (new LaunchDarkly\Client\Api\PersistentStoreIntegrationsBetaApi(config: $config))->getBigSegmentStoreIntegration(
        project_key: null,
        environment_key: null,
        integration_key: null,
        integration_id: null,
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling PersistentStoreIntegrationsBetaApi#getBigSegmentStoreIntegration: {$e->getMessage()}";
}

File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/GetBigSegmentStoreIntegrationExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class GetBigSegmentStoreIntegrationExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        try
        {
            var response = new PersistentStoreIntegrationsBetaApi(config).GetBigSegmentStoreIntegration(
                projectKey: null,
                environmentKey: null,
                integrationKey: null,
                integrationId: null
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling PersistentStoreIntegrationsBetaApi#GetBigSegmentStoreIntegration: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
